src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderDatesResourceTest.java

```diff
@@ -205,29 +205,29 @@ public void testGetOffenderKeyDatesForOffenderSentCalcId() {
-        assertEquals(null,nomisCalculations.getReleaseOnTemporaryLicenceDate());
-        assertEquals(null,nomisCalculations.getAutomaticReleaseDate());
-        assertEquals(null,nomisCalculations.getLateTermDate());
-        assertEquals(null,nomisCalculations.getPostRecallReleaseDate());
-        assertEquals(null,nomisCalculations.getTariffDate());
-        assertEquals(null,nomisCalculations.getEffectiveSentenceEndDate());
-        assertEquals(null,nomisCalculations.getDtoPostRecallReleaseDate());
-        assertEquals(null,nomisCalculations.getEarlyRemovalSchemeEligibilityDate());
-        assertEquals(null,nomisCalculations.getTariffExpiredRemovalSchemeEligibilityDate());
-        assertEquals(null,nomisCalculations.getTopupSupervisionExpiryDate());
-        assertEquals(null,nomisCalculations.getNonParoleDate());
-        assertEquals(LocalDate.of(2022, 10, 20),nomisCalculations.getSentenceExpiryDate());
-        assertEquals(LocalDate.of(2021, 9, 24),nomisCalculations.getLicenceExpiryDate());
-        assertEquals(LocalDate.of(2021, 3, 25),nomisCalculations.getMidTermDate());
-        assertEquals(LocalDate.of(2021, 2, 28),nomisCalculations.getEarlyTermDate());
-        assertEquals(LocalDate.of(2021, 1, 2),nomisCalculations.getHomeDetentionCurfewApprovedDate());
-        assertEquals(LocalDateTime.of(2017, 9, 2, 0,0),nomisCalculations.getCalculatedAt());
-        assertEquals(LocalDate.of(2020, 12, 30),nomisCalculations.getHomeDetentionCurfewEligibilityDate());
+        assertThat(nomisCalculations.getReasonCode()).isEqualTo("NEW");
+        assertThat(nomisCalculations.getComment()).isEqualTo(null);
+        assertThat(nomisCalculations.getParoleEligibilityDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getApprovedParoleDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getConditionalReleaseDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getReleaseOnTemporaryLicenceDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getAutomaticReleaseDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getLateTermDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getPostRecallReleaseDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getTariffDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getEffectiveSentenceEndDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getDtoPostRecallReleaseDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getEarlyRemovalSchemeEligibilityDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getTariffExpiredRemovalSchemeEligibilityDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getTopupSupervisionExpiryDate()).isEqualTo(null);
+        assertThat(nomisCalculations.getNonParoleDate()).isEqualTo(null);
+        assertThat(20)).isCloseTo(LocalDate.of(2022, 10, within(nomisCalculations.getSentenceExpiryDate()));
```

Comment:
now using `assertThat` we find that people have put the parameters the wrong way round 🤷